tests/cephfs/CEPH-10625_11225.py

```diff
@@ -244,34 +244,22 @@ def run(ceph_cluster, **kw):
                         (client_info['mounting_dir'], dir_name, file_name))
         for num in range(0, 5):
             for client in client_info['fuse_clients']:
-                client.exec_command(
-                    cmd="sudo %s %s'%s' --fop create -t %s "
-                        "--multi -b 10 -d 10 -n 10 -T 10 "
-                        "--random --min=1K --max=%dK" %
-                        ("python3 /home/cephuser/Crefi/crefi.py",
-                         client_info['mounting_dir'],
-                         dir_name, 'text', 5),
-                    long_running=True)
-                for i in range(0, 6):
+                for i in range(0, 5):
                     ops = [
                         'create',
-                        'rename',
+                        'setxattr',
+                        'getxattr',
                         'chmod',
-                        'chown',
-                        'chgrp',
-                        'setxattr']
-                    rand_ops = random.choice(ops)
-                    ftypes = ['text', 'sparse', 'binary', 'tar']
-                    rand_filetype = random.choice(ftypes)
-                    rand_count = random.randint(2, 10)
+                        'rename']
                     client.exec_command(
-                        cmd='sudo %s %s%s --fop %s -t %s '
-                            '--multi -b 10 -d 10 -n 10 -T 10 '
-                            '--random --min=1K --max=%dK' %
-                            ("python3 /home/cephuser/Crefi/crefi.py",
-                             client_info['mounting_dir'],
-                             dir_name, rand_ops, rand_filetype, rand_count),
-                        long_running=True)
+                        cmd='sudo python3 smallfile/smallfile_cli.py '
+                            '--operation %s --threads 10 --file-size 4'
+                            '--file-size 4 --files 1000 '
```

Comment:
`--file-size 4` option is being provided twice... I think it is incorrect. At some point of time, you may want to start using format strings

`cmd = "sudo python3 smallfile/smallfile_cli_py --operation {ops[i]} --threads 10 --file-size 4 --files 1000 --files-per-dir 10 --dirs-per-dir 2 --top {client_info[mounting_dir]}{dir_name}`